static/theme-flex/style.css

```diff
@@ -2662,7 +2662,7 @@ a[href*="#no-click"], img[src*="#no-click"] {
         flex: 1 0 auto;
     }
     article section.page .main-content .highlight {
-        display: block;
+        /* display: block; */
```

Comment:
@HarunD I looked at the staging site in IE and it looks great. I just found that links are pushed onto their own line. This change fixed it. What do you think?

Reply:
This change looks good, it seems `display: block` is not needed.